contentcuration/contentcuration/frontend/shared/views/files/mixins.js

```diff
@@ -0,0 +1,63 @@
+import { mapGetters } from 'vuex';
```

Comment:
Was this copied from somewhere else? Wonder if it makes sense to remove the original file too so it's more consolidated

Reply:
I think this was just a mistake in rebase.